uniprot-config/src/main/resources/search-fields-config/uniparc-search-fields.json

```diff
@@ -159,7 +159,7 @@
     "dataType": "INTEGER",
     "sortFieldId": "length_sort",
     "description": "Search by length",
-    "example": "[100 TO 300]",
+    "example": "[50 TO 100]",
```

Comment:
isnt it [1 TO 100] on spread sheet?

Reply:
I tried to reduce a bit to a lighter query with less results